### PR TITLE
Org settings API: Third-Party Admin setting endpoints

### PR DESCRIPTION
**Pull request by rickyyang-okta**
## Description:
- Added documents for new third party admin setting endpoint for orgs
- **Is this PR related to a Monolith release?** Yes. Tentatively 2023.08.0

### Resolves:

* [OKTA-621427](https://oktainc.atlassian.net/browse/OKTA-621427)
image
image
image
image

### CODE REVIEW

**Review by vasishtajayanti-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 1]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
 }
 ```
 
+## Org Thirdparty Admin Settings
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1149]:
```suggestion
## Org Third Party Admin Settings operations
```

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 2]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
...
+The Thirdparty Admin settings API has the following CRUD operation:
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1150]:
```suggestion
The Third-Party Admin settings API has the following CRUD operations:
```

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 3]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
...
+
+* [Get Thirdparty Admin Settings](#get-thirdparty-admin-setting)
+* [Update Thirdparty Admin Settings](#update-thirdparty-admin-setting)
+
+### Get Thirdparty Admin setting
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1155]:
```suggestion
### Get Third-Party Admin settings
```

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 4]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
 }
 ```
 
+## Org Thirdparty Admin Settings
+The Thirdparty Admin settings API has the following CRUD operation:
+
+* [Get Thirdparty Admin Settings](#get-thirdparty-admin-setting)
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1152]:
```suggestion
* [Get Third-Party Admin settings](#get-third-party-admin-settings)
```

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 5]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
...
+* [Update Thirdparty Admin Settings](#update-thirdparty-admin-setting)
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1153]:
```suggestion
* [Update Third-Party Admin settings](#update-third-party-admin-settings)
```

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 6]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
...
+
+### Get Thirdparty Admin setting
+
+<ApiOperation method="get" url="/api/v1/org/orgSettings/thirdPartyAdminSetting" />
+
+Gets the third party admin setting for the current org. The default value is false.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1159]:
```suggestion
Gets the Third-Party Admin setting for the current Org. The default value is `false`.
```

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 7]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
...
+#### Response body
+
+| Property | Type | Description |
+| ------------------ | ------------------------------ | ----------------------------------------------------------------------------------------------------------------- |
+| `thirdPartyAdmin` | boolean | The current third party admin setting for the org|
+
+
+#### Use example
+
+This request returns the third party admin setting for the current org
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1170]:
```suggestion
This request returns the third-party admin setting for the current org.
```

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 8]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
...
+"https://${yourOktaDomain}/api/v1/org/orgSettings/thirdPartyAdminSetting"
+```
+
+##### Response
+```http
+{
+    "thirdPartyAdmin": false,
+}
+```
+### Update Thirdparty Admin setting
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1188]:
```suggestion
### Update Third-Party Admin setting
```

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 9]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
...
+
+<ApiOperation method="post" url="/api/v1/org/orgSettings/thirdPartyAdminSetting" />
+
+updates the third party admin setting for the current org
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1192]:
```suggestion
Updates the Third-Party Admin setting for the current Org
```

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 10]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
...
+#### Response body
+
+| Property | Type | Description |
+| ------------------ | ------------------------------ | ----------------------------------------------------------------------------------------------------------------- |
+| `thirdPartyAdmin` | boolean | The third party admin setting to update for the org|
+
+
+#### Use example
+
+This request updates the third party admin setting for the current org
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1203]:
```suggestion
This request updates the third-party admin setting for your org.
```

**Review by brentschaus-okta — APPROVED** [review]
LGTM!

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 11]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
 }
 ```
 
+## Org Third-Party Admin setting operations
+The Third-party Admin setting API has the following CRUD operations:
+
+* [Get the Third-party Admin setting](#get-the-thirdparty-admin-setting)
+* [Update the Third-party Admin Setting](#update-the-thirdparty-admin-setting)
+
+### Get the Third-party Admin setting
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1155]:
```suggestion
### Get the Third-Party Admin setting
```

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 12]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
...
+
+<ApiOperation method="get" url="/api/v1/org/orgSettings/thirdPartyAdminSetting" />
+
+Gets the Third-party Admin setting. The default value is `false`.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1159]:
```suggestion
Gets the Third-Party Admin setting. The default value is `false`.
```

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 13]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
...
+"https://${yourOktaDomain}/api/v1/org/orgSettings/thirdPartyAdminSetting"
+```
+
+##### Response
+```http
+{
+    "thirdPartyAdmin": false,
+}
+```
+### Update the Third-party Admin setting
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1188]:
```suggestion
### Update the Third-Party Admin setting
```

**Review by williamngo-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 14]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
 }
 ```
 
+## Org Third-Party Admin setting operations
+The Third-party Admin setting API has the following CRUD operations:
```
Comment (williamngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1150]: Should this be "Third-Party" for consistency on the rest of the doc

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 15]
```diff
@@ -1146,6 +1146,82 @@ HTTP/1.1 400 Bad Request
...
+
+* [Get the Third-party Admin setting](#get-the-thirdparty-admin-setting)
```
Comment (williamngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:1152]: Mix of Third-party and Third-Party

**Pull request merged**